src/test/abilities/stall.test.ts

```diff
@@ -0,0 +1,95 @@
+import { MovePhase } from "#app/phases";
+import GameManager from "#test/utils/gameManager";
+import { getMovePosition } from "#test/utils/gameManagerUtils";
+import { Abilities } from "#enums/abilities";
+import { Moves } from "#enums/moves";
+import { Species } from "#enums/species";
+import Phaser from "phaser";
+import { afterEach, beforeAll, beforeEach, describe, expect, it } from "vitest";
+
+
+describe("Ability - Stall", () => {
```

Comment:
nit: to be consistent with other test titles
```suggestion
describe("Abilities - Stall", () => {
```